speeduino/decoders.cpp

```diff
@@ -500,6 +500,15 @@ void triggerPri_missingTooth(void)
       {
         bool isMissingTooth = false;
 
+        if(( (currentStatus.hasSync == false) || (currentStatus.RPM < 2000)))
+        {
+          if ((configPage4.trigPatternSec == SEC_TRIGGER_POLL) && (configPage10.PollLevelTeeth > 1) && (toothCurrentCount == configPage10.PollLevelTeeth)) // at selected tooth check if the cam sensor is high or low in poll level mode
+          {          
+            if (configPage4.PollLevelPolarity == READ_SEC_TRIGGER()) { revolutionOne = 1; }
+            else { revolutionOne = 0; }
```

Comment:
Isn't this the same as `revolutionOne = configPage4.PollLevelPolarity == READ_SEC_TRIGGER();` ?